src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    private static final int PASSWORD_MAX_LENGTH = 10;
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password != null
+                && password.length() >= PASSWORD_MAX_LENGTH
+                && password.equals(repeatPassword)) {
+            return true;
+        }
+        throw new PasswordValidationException("Wrong passwords");
     }
 }
+
+
```

Comment:
please remove redundant empty lines - there should be only 1 in the end of the file